coverage/src/Tooltip-Modal.js

```diff
@@ -0,0 +1,144 @@
+const modalTemplate = document.createElement('template');
+modalTemplate.innerHTML = `
+  <style>
+    .modal {
+      display: none;
+      position: fixed;
+      z-index: 1;
+      left: 0;
+      top: 0;
+      width: 100%;
+      height: 100%;
+      background-color: rgba(0, 0, 0, 0.4);
```

Comment:
Can you make the background clickable so the modal closes when the user clicks it? This is how prod bam.iobio works.